frontend/src/__test__/unit_test/PathLength.test.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
Are these no-empty-function lint exemption needed still as it looks like there are no longer empty functions in a lot of these test files?